cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
```

Comment:
We should follow dependency injection principle in order to replace any dependancies while applying test cases
if any logic is dependent on this object we should inject during navigation